### PR TITLE
[EDU-3172] feat: add automate origin shield

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new guide on How to automate Origin Shield.

[EDU-3172]: https://aziontech.atlassian.net/browse/EDU-3172?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+2. Select the edge firewall in which you want to configure the rule.
+3. Click the **Rules Engine* tab.
+4. Click the **New Rule** button.
+5. Give your rule a name and, optionally, a description.
+6. As a comparison operator, select **does not match**.
+7. As an argument, select the `Azion Origin Shield` list.
+8. In the **Behaviors** section, select **Drop (Close Without Response)** from the behavior list.
+9. Click the **Save** button.
+
+If your application receives a request generated from an IP that isn't in the list, edge firewall will drop the request based.
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx:32]:
```suggestion
If your application receives a request generated from an IP that isn't in the list, the edge firewall will drop the request.
```

Comment (gabriel-azion) [reply src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx:32]: not sure... the original seemed weird

**Review by gabriel-azion — APPROVED** [review]
only one suggestion

File: src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx [hunk 2]
```diff
@@ -0,0 +1,143 @@
+---
+title: Automate Origin Shield
```
Comment (hannah-sk) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx:2]:
```suggestion
title: How to automate Origin Shield
```

File: src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+description: Create a rule to automate the use of the Origin Shield list and block potential harmful IPs.
+meta_tags: edge, secure, origin shield, list, rules engine, behavior
+namespace: docs_secure_automate_origin_shield
+menu_namespace: secureMenu
+permalink: /documentation/guides/secure/automate-origin-shield/
+---
+
+import Button from '~/components/Button.astro'
+
+**Origin Shield** provides a list of IP addresses used in Azion's Edge.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx:12]:
```suggestion
**Origin Shield** provides a list of IP addresses used in Azion's Edge Network.
```

Nodes? Platform?

File: src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+<Button href="/en/documentation/products/secure/secure-infrastructure/" text="go to secure an infrastructure" variant="secondary"></Button>
+
+Using it benefits your application's protection, but as the list is maintained by Azion and the IP addresses in it change from time to time, you can automate a behavior using Rules Engine to guarantee you're always using the updated list.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx:16]:
```suggestion
By activating Origin Shield, your application's is protected from a network list is maintained by Azion. The IP addresses in it change from time to time, but you can automate a behavior using Rules Engine to guarantee you're always using the updated list.
```

File: src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+8. In the **Behaviors** section, select **Drop (Close Without Response)** from the behavior list.
+9. Click the **Save** button.
+
+If your application receives a request generated from an IP that isn't in the list, edge firewall will drop the request based.
+
+---
+
+## Via API
+
+1. Run the following `GET` request to retrieve the `id` of the `Azion Origin Shield` list:
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx:38]:
```suggestion
1. Run the following `GET` request to retrieve the `id` of the **Azion Origin Shield** list:
```

File: src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+    "id": 28629,
+    "order": 3
+  },
+  "schema_version": 3
+}
+```
+
+5. Wait a few minutes for the changes to propagate.
+
+If your application receives a request generated from an IP that isn't in the list, edge firewall will drop the request based.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx:143]: Contributors!

File: src/content/docs/en/pages/secure-journey/secure-infrastructure.mdx [hunk 7]
```diff
@@ -21,6 +21,7 @@ To access the Origin Shield list, you must have:
 
 - An [edge firewall created](/en/documentation/products/secure/secure-application/).
 - The [Network Layer Protection module](/en/documentation/products/guides/secure/configure-main-settings/#modules) enabled.
+- Be subscribe to the Origin Shield add-on. Contact the [sales team]() to subscribe to it.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/secure-infrastructure.mdx:24]:
```suggestion
- Be subscribe to the Origin Shield add-on. Contact the [sales team](https://www.azion.com/en/contact-sales/) to subscribe to it.
```

**Review by bru-andrade — COMMENTED** [review]
Alguns comentários

File: src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx (hunk 4 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx:16]:
```suggestion
By activating Origin Shield, your application is protected from a network list maintained by Azion. The IP addresses in it change from time to time, but you can automate a behavior using Rules Engine to guarantee you're always using the updated list.
```

File: src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+
+import Button from '~/components/Button.astro'
+
+**Origin Shield** provides a list of IP addresses used in Azion's Edge.
+
+<Button href="/en/documentation/products/secure/secure-infrastructure/" text="go to secure an infrastructure" variant="secondary"></Button>
+
+Using it benefits your application's protection, but as the list is maintained by Azion and the IP addresses in it change from time to time, you can automate a behavior using Rules Engine to guarantee you're always using the updated list.
+
+## Via RTM
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx:18]:
```suggestion

---

## Via RTM
```

File: src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx (hunk 6 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx:143]:
```suggestion
If your application receives a request generated from an IP that isn't in the list, the edge firewall will drop the request.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx:143]: para alinhar com um comentário anterior

**Review by MarianaAguilera — APPROVED** [review]
some suggestions

File: src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx (hunk 3 above)
Comment (gabriel-azion) [reply src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx:12]: ou **Origin Shield** provides a list of Azion Edge Nodes' IP addresses.

**Pull request merged**